db.go

```diff
@@ -1822,19 +1822,22 @@ func (db *DB) BannedNamespaces() []uint64 {
 // KVList contains a list of key-value pairs.
 type KVList = pb.KVList
 
-// Subscribe can be used to watch key changes for the given key prefixes.
+// Subscribe can be used to watch key changes for the given key prefixes and the ignore string.
 // At least one prefix should be passed, or an error will be returned.
 // You can use an empty prefix to monitor all changes to the DB.
+// Ignore string is the byte ranges for which prefix matching will be ignored.
+// For example: ignore = "2-3", and prefix = "abcde" will match for keys "abxxe", "abdfe" etc.
 // This function blocks until the given context is done or an error occurs.
 // The given function will be called with a new KVList containing the modified keys and the
 // corresponding values.
-func (db *DB) Subscribe(ctx context.Context, cb func(kv *KVList) error, prefixes ...[]byte) error {
+func (db *DB) Subscribe(ctx context.Context, cb func(kv *KVList) error, ignore string,
```

Comment:
Can't we use the pb.Match here directly? There is a list of prefixes. What if I want to specify multiple ignore strings? Right now you're using the same one for all prefixes.

Also, this is a breaking change (Public API change)

Reply:
Yeah. That would be better to have. Thanks